src/lib/components/Feedback/index.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * @fileoverview Element that handles feedback on posts.
+ */
+import {html} from 'lit-element';
+import {BaseElement} from '../BaseElement';
+import {trackEvent} from '../../analytics';
+import './_styles.scss';
+
+/**
+ * Catches feedback submission on posts to submit to Analytics.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class Feedback extends BaseElement {
+  constructor() {
+    super();
+    this.submitted = false;
+    this.submit = this.submit.bind(this);
+  }
+
+  submit(event) {
+    event.preventDefault();
+    const form = new FormData(event.target);
+    const formIterable = Array.from(form.entries());
+    if (!formIterable.length || this.submitted) {
+      return;
+    }
+
+    this.submitted = true;
+    this.requestUpdate();
+
+    formIterable.forEach((entry) => {
+      trackEvent({
+        category: 'Feedback',
+        action: 'submit',
+        label: entry[0],
+        value: Number(entry[1]),
+      });
+    });
+  }
+
```

Comment:
```suggestion
updated() {
    if (this.submitted) {
      this.querySelector('.w-feedback__confirmation').focus();
    }
  }
```

Use this to focus the confirmation text